package.json

```diff
@@ -141,5 +141,8 @@
   },
   "engines": {
     "node": ">=20.x"
+  },
+  "volta": {
+    "node": "20.10.0"
```

Comment:
The addition of `volta` is a great change IMO. 
- Please also add the version of npm we are using to volta.
- Update the developer documentation explaining what is `volta` and how the use may opt into the same. 
- You may do this in a new issue. If chosen to do so, then remove this change from this PR.

Reply:
@EshaanAgg sure, will do this in a new issue. Any views on this @palisadoes ?

Reply:
Please work with Eshaan on this and future work related to this. We delegate reviewers for this purpose. I am not necessary

Reply:
Okay. Will work on this with Eshaan.